### ISSUE TITLE
Suggestion: Manage all active / inactive widgets for Network

### ISSUE THREAD

**Issue by ingageco** [siteorigin/siteorigin-panels#665]
It would be great if we could activate / deactivate widgets on a network-wide basis. Im currently using the plugin on a network and having to go to each site and activate when i create a new widget to be available for all sites.

**Comment by Misplon** [comment]
Hi ingageco, thanks for posting :) 

Should we move this to the Widgets Bundle repo? Are you activating widgets at `Plugins > SiteOrigin Widgets`?

**Comment by ingageco** [comment]
Yes i am - I wasn't sure if this should be a feature of the plugin or of the widgets bundle!
